Makefile

```diff
@@ -0,0 +1,11 @@
+test:
+	@echo Running tests
+	@go test $$(go list ./... | grep -v /vendor/)
```

Comment:
Since we expect Golang 1.9.x to be used, excluding vendor should not be necessary anymore. So a `go test ./...` here should suffice and will be quicker.

Reply:
Right, I will remove it